integration-healthcheck/src/main/java/org/symphonyoss/integration/healthcheck/services/ServiceVersion.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2016-2017 Symphony Integrations - Symphony LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.symphonyoss.integration.healthcheck.services;
+
+/**
+ * Represents the service version.
+ * Created by rsanchez on 27/01/17.
+ */
+public class ServiceVersion implements Comparable<ServiceVersion> {
```

Comment:
Did you look for an existing library that models "semantic versioning"?  Something like https://github.com/zafarkhaja/jsemver